server/datastore/datastore.go

```diff
@@ -2,14 +2,13 @@
 // by the database.
 package datastore
 
-import (
-	"time"
-)
+import "time"
 
 // DB is an interface for the app database.
 // Any Struct used as the app database must implement these functions.
 type DB interface {
-	Init(config DatabaseConfig) error // Initialize pgx connection
+	Init(config DatabaseConfig) error                      // Initialize pgx connection
```

Comment:
I know this isn't your change but could you remove this comment? we shouldn't have a reference to a database specific thing in the interface